bigbio/hub/hub_repos/ggponc2/README.md

```diff
@@ -0,0 +1,63 @@
+---
+language:
+  - de 
+bigbio_language:
+  - German
+multilinguality: monolingual
+pretty_name: GGPONC2
+homepage: https://www.leitlinienprogramm-onkologie.de/projekte/ggponc-english/
+bigbio_pubmed: false
+bigbio_public: true
```

Comment:
if this is a private dataset then `bigbio_public` should be false

Reply:
You are very welcome! All changes have been implemented. The test log is on the comment just below.